exploration/MarkerStateStamped[] active_states
